week-03/README.md

```diff
@@ -16,69 +16,56 @@ The instructor will then review the pull request and make comments should furthe
 
 
 [Course Textbook Chapter 3, Problem 1]
-
-
+-Question(tchan49-stat6250): What can you do to make your SAS statement consistent and readable? 
+-Answer(tchan49-stat6250): a)begin DATA and PROC steps in column one, b) indent statements within step, c) begin RUN statements in column one, d) include a RUN statement after every DATA step or PROC steps
 
 [Course Textbook Chapter 3, Problem 2]
-
-
+-Question(tchan49-stat6250): Before resubmitting your updated SAS statement, which window should you double-check? 
 
 [Course Textbook Chapter 3, Problem 3]
+-Question(tchan49-stat6250): When you miss a quotation in a SAS statement, where can you find the error message? 
 
-
- 
 [Course Textbook Chapter 3, Problem 4]
-
-
+-Question(tchan49-stat6250): When you have a syntax error, can you check the error message at the Output window?
 
 [Course Textbook Chapter 3, Problem 5]
-
-
+-Question(tchan49-stat6250): Do you get syntax error when some data values are not appropriate for the SAS statement? 
 
 [Course Textbook Chapter 3, Problem 6]
+-Question(tchan49-stat6250): When you have an invalid option in SAS program, is it a syntax error or data error? 
 
-
-
-[Course Textbook Chapter 3, Problem 7]
-
-
+[Course Textbook Chapter 3, Problem 7] 
+-Question(tchan49-stat6250): How can you correct the misspelling key work on a SAS statement? Do you need to re-type the entire statement? 
 
 [Course Textbook Chapter 3, Problem 10]
-
-
+-Question(tchan49-stat6250): If you miss the RUN statement for a DATA step, what is going to display on the active window? 
+-Answer(tchan49-stat6250): "DATA step running" for a long time. 
 
 [Course Textbook Chapter 4, Problem 1]
-
-
+-Question(tchan49-stat6250): How can you replace Obs column with other columns? 
+-Answer(tchan49-stat6250): Use ID statement 
 
 [Course Textbook Chapter 4, Problem 3]
-
-
+-Question(tchan49-stat6250): When you test for multiple values of the same variable, besides using OR in a WHERE statement, what operator can you use? 
 
 [Course Textbook Chapter 4, Problem 4]
+-Question(tchan49-stat6250): Do you create a temporary dataset before identifying the datatset that you want to read or after? 
 
+[Course Textbook Chapter 4, Problem 7] 
+-Question(tchan49-stat6250): Is BY statement required when you try to sort a dataset? 
 
+ [Course Textbook Chapter 4, Problem 9]
+ -Question(tchan49-stat6250): What is the difference between AND and OR operator? How are they used differently when testing variables from the same attribute or different attributes? 
 
-[Course Textbook Chapter 4, Problem 7]
-
-
-
-[Course Textbook Chapter 4, Problem 9]
-
-
-
-[Course Textbook Chapter 4, Problem 10]
-
-
+[Course Textbook Chapter 4, Problem 10] 
```

Comment:
Existing content edited; please remove spaces added after problem headers (after closing square braces) and DM me in Slack with the PR URL for re-evaluation